### DISCUSSION TITLE
Configure suggestions to update pull request branches

### DISCUSSION THREAD

**Discussion by hanzzzzz** [community/community#111221] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Regarding [this](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-suggestions-to-update-pull-request-branches) feature:

It would be great to be able to configure the options on how to update the pull request branch. The repository general setting `Always suggest updating pull request branches` should be extended to allow selecting if only `Update`, only `Rebase` or both options should be available. This would help misuse of the feature in repositories where only one of the two options should be used.

**Comment by gdvalderrama** [comment]
see [this existing discussion](https://github.com/orgs/community/discussions/12032)
